### PR TITLE
Update Windows packaging for per-mod launchers.

### PR DESCRIPTION
**Pull request by pchote**
This updates the windows packaging to install separate launchers and shortcuts for each mod, as well as some other much needed changes:

* Release/playtest/dev installs are treated separately and can now be installed side by side.
* Removed the readme and uninstaller links from the start menu to conform with [best practices](http://nsis.sourceforge.net/Best_practices#Start_Menu.2FStart_Screen).
* Includes additional metadata (version, install size, readme, quiet uninstall command) for the control panel software list.

~~Depends on #13049.~~

Test build (fake playtest): [OpenRA-playtest-20170415.exe](https://www.dropbox.com/s/e2jqjkgubymblgl/OpenRA-playtest-20170415.exe?dl=0).
Test build (dev version): See Appveyor artefact page.
The two test builds can be installed together to test the side-by-side version support.

### CODE REVIEW

**Comment by pchote** [comment]
Rebased after #13049.

**Review by phrohdoh — APPROVED** [review]
LGTM

pinging @abcdefg30 and/or @reaperrr

**Comment by reaperrr** [comment]
I'm too tired to review this now, I'll do so tomorrow unless someone beats me to it.

File: packaging/windows/OpenRA.nsi [hunk 1]
```diff
@@ -18,6 +18,7 @@
 !include "MUI2.nsh"
 !include "FileFunc.nsh"
 !include "WordFunc.nsh"
+!include "FileFunc.nsh"
```
Comment (reaperrr) [inline packaging/windows/OpenRA.nsi]: This is already listed 2 lines above, is this really needed twice?

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
All launchers work, side-by-side installs work, installation and deinstallation (from system control) works.

My question on that duplication aside, I consider this mergable.

**Review by abcdefg30 — APPROVED** [review]

File: packaging/windows/OpenRA.nsi (hunk 1 above)
Comment (pchote) [inline packaging/windows/OpenRA.nsi]: Nope, that was an oversight on my part.  Fixed.

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/9a292cbf5f1931605de642f6e2e73e6bf2c412b5

**Pull request merged**